internal/server/edit_lay_percentage.go

```diff
@@ -0,0 +1,73 @@
+package server
+
+import (
+	"net/http"
+	"strconv"
+
+	"github.com/ministryofjustice/opg-sirius-user-management/internal/sirius"
+)
+
+type EditLayPercentageClient interface {
+	RandomReviews(sirius.Context) (sirius.RandomReviews, error)
+	EditLayPercentageReviewCycle(ctx sirius.Context, layPercentage string, reviewCycle string) (error)
+}
+
+type editLayPercentageVars struct {
+	Path                string
+	XSRFToken           string
+	LayPercentage       string
+	ReviewCycle         string
+	Errors              sirius.ValidationErrors
+	Error               string
+}
+
+func editLayPercentage(client EditLayPercentageClient, tmpl Template) Handler {
```

Comment:
Don't know if it is an issue with github displaying it but the formatting is really off in this function

Reply:
go fmt ran